soul-bootstrap/src/main/resources/application-local.yml

```diff
@@ -43,7 +43,7 @@ soul:
         websocket :
              urls: ws://localhost:9095/websocket
     exclude:
-      enable: false
+      enabled: false
```

Comment:
is this enabled？  can you move this config？

Reply:
Has moved, Conflict resolved